aws/data_source_aws_availability_zones.go

```diff
@@ -59,16 +64,38 @@ func dataSourceAwsAvailabilityZonesRead(d *schema.ResourceData, meta interface{}
 		return fmt.Errorf("Error fetching Availability Zones: %s", err)
 	}
 
-	raw := make([]string, len(resp.AvailabilityZones))
-	for i, v := range resp.AvailabilityZones {
-		raw[i] = *v.ZoneName
-	}
+	azs := resp.AvailabilityZones
+	sort.Sort(availabilityZones(azs))
```

Comment:
We can use `sort.Slice()` instead of defining our own sortable type. 😄 

```go
sort.Slice(resp.AvailabilityZones, func(i, j int) bool {
	return aws.StringValue(resp.AvailabilityZones[i].ZoneName) < aws.StringValue(resp.AvailabilityZones[j].ZoneName)
})
```

Reply:
Done.